<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingAlignmentProjectionForm</class>
 <widget class="QWidget" name="SettingAlignmentProjectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditItemName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAngle">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Usage layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Peak count</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxUsageLayer">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPeakCount">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set to all points</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetOne">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set in this point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameLineGraph">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1031</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Abandon rate Low</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAbandonRateL">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Abandon rate High</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAbandonRateH">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxPeakIsolation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Peak Isolation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
